portal-api/api-tests/wipeDB.js

```diff
@@ -1,10 +1,11 @@
 const db = require('../src/drivers/db-client');
 
 const wipe = async (collections) => {
-  const drop = async (collection) => new Promise(async (resolve, reject) => {
+  // eslint-disable-next-line no-async-promise-executor
```

Comment:
I think this might be able to be fixed in the same way as the other `wipeDB` file

Reply:
Have refactored,.